rules_poetry/defs.bzl

```diff
@@ -110,29 +89,17 @@ COMMON_ARGS = [
 ]
 
 def _download(ctx, requirements):
-    destination = ctx.actions.declare_directory("wheels/%s" % ctx.attr.name)
+
     toolchain = ctx.toolchains["@bazel_tools//tools/python:toolchain_type"]
     runtime = toolchain.py3_runtime
+    inputs = depset([requirements], transitive = [runtime.files])
+    tools = depset(direct = [runtime.interpreter], transitive = [runtime.files])
 
-    # The Python interpreter can be either provided through a path
-    # (platform runtime), or through a label (in-build runtime).
-    # See https://docs.bazel.build/versions/master/be/python.html#py_runtime
-    if runtime.interpreter_path != None:
-        executable = runtime.interpreter_path
-        inputs = depset([requirements])
-        tools = depset(direct = [ctx.executable._pip])
-    else:
-        executable = runtime.interpreter.path
-        inputs = depset([requirements], transitive = [runtime.files])
-        tools = depset(direct = [runtime.interpreter,
-                                 ctx.executable._pip], transitive = [runtime.files])
-
-    pip_path = ctx.executable._pip.path
+    python = ctx.toolchains["@bazel_tools//tools/python:toolchain_type"].py3_runtime
+    destination = ctx.actions.declare_directory("wheels/%s" % ctx.attr.name)
```

Comment:
Does this declare_directory still cause issues?

Reply:
no even though its declared (it has to be to install the files to the location), the return output from the rule is just the zipfile